homeassistant/components/config/config_entries.py

```diff
@@ -47,13 +49,22 @@ class ConfigManagerEntryIndexView(HomeAssistantView):
     async def get(self, request):
         """List flows in progress."""
         hass = request.app['hass']
+
+        def supports_options(domain):
+            """"""
+            if domain == 'axis':
```

Comment:
?

Reply:
Sorry, that is just due to my dev environment having some stuff laying around from the axis config entry migration. It will be removed